includes/payment-methods/class-wc-gateway-stripe-sepa.php

```diff
@@ -117,6 +118,8 @@ public function __construct() {
 
 		add_action( 'woocommerce_update_options_payment_gateways_' . $this->id, array( $this, 'process_admin_options' ) );
 		add_action( 'wp_enqueue_scripts', array( $this, 'payment_scripts' ) );
+        
+		add_action( 'woocommerce_order_status_changed', array( $this, 'maybe_activate_subscriptions_early'), 10, 4 );
```

Comment:
I think we should use the [`woocommerce_subscription_status_on-hold`](https://docs.woocommerce.com/document/subscriptions/develop/action-reference/) action here to narrow the scope of when we change the subscription status.

Reply:
Let's not do that; see https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1400#discussion_r563503004